builds/mariner/SPECS/aziot-edge/aziot-edge.spec

```diff
@@ -0,0 +1,191 @@
+%define iotedge_user iotedge
+%define iotedge_group %{iotedge_user}
+%define iotedge_home %{_localstatedir}/lib/aziot/edged
+%define iotedge_logdir %{_localstatedir}/log/aziot/edged
+%define iotedge_socketdir %{_localstatedir}/lib/iotedge
+%define aziot_confdir %{_sysconfdir}/aziot
+%define iotedge_confdir %{aziot_confdir}/edged
+
+Name:           aziot-edge
+Version:        @@VERSION@@
+Release:        3%{?dist}
```

Comment:
What's the value of the `3%` ? Should it be `@@RELEASE@@` or some such?

Reply:
changed to @@RELEASE@@